counterpartylib/lib/messages/issuance.py

```diff
@@ -124,6 +137,36 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
         if destination:
             problems.append('cannot transfer a non‐existent asset')
 
+    # validate parent ownership for subasset
```

Comment:
sure, I'm fine leaving it out on reissuance descriptions if we can be strict about it and say that ;;l exists on the initial issuance to tie that longname to the numerical asset, but reissuances must not have it (and fail if they do). That will also fail if ;;l is specified and does not match the initial longname as well, of course.

Can you also open up a PR to CIP004 that fills in that spec with these clarifications? (that is, given the decisions we are making here, the CIP is lacking necessary detail in certain areas)